docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -58,9 +59,9 @@ To install the Azure CLI using PowerShell, start PowerShell **as administrator**
    $ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
    ```
 
-This will download and install the latest version of the Azure CLI for Windows. If you already have a version installed, the installer will update the existing version.
+This will download and install the latest 32-bit installer of the Azure CLI for Windows. If you prefer 64-bit one, please change URL to `https://aka.ms/installazurecliwindowsx64`. If you already have a version installed, the installer will update the existing version.
 
-To install specific version, replace the `-Uri` argument with `https://azcliprod.blob.core.windows.net/msi/azure-cli-<version>.msi` with version segment changed. Available versions can be found at [Azure CLI release notes](./release-notes-azure-cli.md).
+To install specific version, replace the `-Uri` argument with `https://azcliprod.blob.core.windows.net/msi/azure-cli-<version>.msi` (32-bit) or `https://azcliprod.blob.core.windows.net/msi/azure-cli-<version>-x64.msi` (64-bit) with version segment changed. Available versions can be found at [Azure CLI release notes](./release-notes-azure-cli.md). (64-bit MSI is available from 2.51.0.)
```

Comment:
```suggestion
To install a specific version, replace the `-Uri` argument with the URL described in [Specific version](#specific-version).  Here is an example of using the 32-bit installer of the Azure CLI version [2.50.0](/cli/azure/release-notes-azure-cli#july-04-2023) in PowerShell:

   ```PowerShell
   $ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://azcliprod.blob.core.windows.net/msi/azure-cli-2.50.0-x32.msi -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
   ```
```

Reply:
Could you please remove the `-x32` suffix and I can merge this suggestion.

Reply:
`[Specific version](#specific-version)` is under `# [Microsoft Installer (MSI)](#tab/azure-cli)` tab, but not available under `# [Microsoft Installer (MSI) with Command](#tab/powershell)`, so the link will not be clickable.